mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,383 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'RegionResizeRequest'
+min_version: beta
+description: |
+  Represents a Regional Managed Instance Group Resize Request
+
+  Resize Requests are the Managed Instance Group implementation of Dynamic Workload Scheduler Flex Start.
+
+  With Dynamic Workload Scheduler in Flex Start mode, you submit a GPU capacity request for your AI/ML jobs by indicating how many you need, a duration, and your preferred region. Dynamic Workload Scheduler intelligently persists the request; once the capacity becomes available, it automatically provisions your VMs enabling your workloads to run continuously for the entire duration of the capacity allocation.
+references:
+  guides:
+    'ABOUT_RESIZE_REQUEST_IN_MIG': 'https://cloud.google.com/compute/docs/instance-groups/about-resize-requests-mig'
+  # Link to the REST API reference for the resource.
+  api: 'https://cloud.google.com/compute/docs/reference/rest/beta/regionInstanceGroupManagerResizeRequests'
+docs:
+### List Method ###
+base_url: 'projects/{{project}}/regions/{{region}}/instanceGroupManagers/{{instance_group_manager}}/resizeRequests'
+### Get Method
+self_link: 'projects/{{project}}/regions/{{region}}/instanceGroupManagers/{{instance_group_manager}}/resizeRequests/{{name}}'
+immutable: true
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+# Sets parameters for handling operations returned by the API.
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  # Overrides which API calls return operations. Default: ['create',
+  # 'update', 'delete']
+  # actions: ['create', 'update', 'delete']
+  operation:
+    base_url: '{{op_id}}'
+
+### Update method ###
+# Resize requests are currently not update-able
+
+### Delete Method ###
+# Custom delete method to handle resize request cancellations vs. deletions.
+# If a resize request is in the CREATING or ACCEPTED state, it must be canceled before it can be
+# deleted. If a resize request is NOT in any of the mentioned state, it can be directly deleted.
+custom_code:
+  custom_delete: 'templates/terraform/custom_delete/compute_rmig_resize_request_delete.go.tmpl'
+# Examples for testing
+examples:
+  - name: 'compute_rmig_resize_request'
+    min_version: beta
+    primary_resource_id: 'a3_resize_request'
+    vars:
+      resize_request_name: 'a3-dws'
+# Resize request parameters injected via URL
+parameters:
+  - name: 'region'
+    type: ResourceRef
+    description: |
+      Name of the compute region scoping this request. Name should conform to RFC1035.
+    url_param_only: true
+    required: true
+    resource: 'Region'
+    imports: 'name'
+  - name: 'instanceGroupManager'
+    type: ResourceRef
+    description: |
+      The name of the regional managed instance group. The name should conform to RFC1035 or be a resource ID.
```

Comment:
I don't think "name" is correct here, we want a ResourceReference. 
I would rephrase this.

Reply:
Done! Check please.